app/views/pledgegroups/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="pledge-groups-container">
+    <h3>Pledge Groups</h3>
+
+    <%= button_to "New Pledge Group", new_pledgegroup_path, :method => "get" %>
+
+    <div class="pledge-group-list">  
+     <table>
+        <thead>
+            <tr>
+                <th>ID</th>
+                <th>admins</th>
+                <th>public</th>
+                <th>pledges</th>
+                <th>budget</th>
+                <th>amount paid</th>
+            </tr>
+        </thead>
+
+        <tbody>
+            <% @pledgegroups.each do |pledgegroup| %>
+            <tr>
+                <td><%= pledgegroup.id %></td>
+                
+                <td>
+                    <%= pledgegroup.admins.each do |user| %>
+                        <p><%= user.name %></p>
+                    <% end %>
+                </td>
+
+                <td><%= pledgegroup.is_public %></td>
+                <td><%= pledgegroup.pledges %></td>
```

Comment:
```suggestion
```
Not sure that I'd want to see who's paying what on this page

Reply:
Perhaps we can add this as an admin privilege in the future, that admins can see it - but ok to not have it here for now